### ISSUE TITLE
Change bitcoind download source or add a dialog menu to choose source

### ISSUE THREAD

**Issue by nyxnor** [raspiblitz/raspiblitz#2502]
bitcoincore.org blocks UK clients, so:

1. Github: https://github.com/bitcoin/bitcoin/releases with signed tag:
![image](https://user-images.githubusercontent.com/69700936/131153050-0dee581c-a640-4abf-b83b-a55bdffca645.png)

2. bitcoincore.org onion binary source -> http://6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion/bin

3. Or prefix with `torsocks` this https://github.com/rootzoll/raspiblitz/blob/79ccb8d3433a59ee8268b458436dbb4f3a824aa3/home.admin/config.scripts/bitcoin.update.sh#L122

4. Laanwj onion git -> http://nxshomzlgqmwfwhcnyvbznyrybh3gotlfgis7wkv7iur2yj2rarlhiad.onion/git/bitcoin.git Source -> https://laanwj.github.io/2018/06/08/tor-repository.html
-------------
Probably the project is more inclined to use the trusted signed binaries, so the options 1 and 2 are the most compatible with Blitz, but documenting here the git options 3 and 4.

**Comment by rootzoll** [comment]
Yeah changing to github releases might be the easiest solution for now .. could be combined with the refactor to let the build script call the install scripts to prevent "code duplication".

**Comment by openoms** [comment]
I'd support the fallback to download over Tor if both bitcoincore.org (as in #2527) and the github link fails.
